docs/source/cli_help/github.rst

```diff
@@ -66,6 +66,11 @@ a GitHub API token with ``repo`` scope is required, see https://github.com/setti
 
   Sets the base of the current branch's PR to upstream (parent) branch, as seen by git machete (see ``git machete show up``).
 
+``sync``:
+
+  Synchronizes with the remote repository: removes untracked and unmanaged branches also checkouts open PRs for the current user associated with the Github token,
```

Comment:
s/also/and also

Reply:
Can you alaborate?

Reply:
```suggestion
  Synchronizes with the remote repository: removes untracked and unmanaged branches and also checkouts open PRs for the current user associated with the Github token,
```

Reply:
I've used `sed` syntax instead of a github suggestion ;)

Reply:
Well, google says it should be just `also`:
```
Is it also or and also?
"And" is used to join two clauses in a sentence . "Also" is used to emphasize the addition to the first clause. We ahead a hearty meal and also a delicious one.
```

Reply:
I understand the checking out as the addition to removing the branches, wdyt?

Reply:
Hmmm the example you've posted indicates `and also` indeed:

_We ahead a hearty meal **and** also a delicious one._

Reply:
(btw _we ahead_? 🤔 sounds weird, I'd expect _we've had_ or sth)